.description.md

```diff
@@ -0,0 +1,31 @@
+<!-- Please provide a short overview of the features of this service.-->
+
+This service implements an interface to extract file encryption secrects from a
+[GA4GH Crypt4GH](https://www.ga4gh.org/news/crypt4gh-a-secure-method-for-sharing-human-genetic-data/)
+encrypted file in a HashiCorp Vault and request user-specific file envelopes
+containing these secrets.
+The Vault is owned and controlled by this service, no other service has access to it.
```

Comment:
This is a integration detail and not enforced in the code base, maybe better to remove.
```suggestion
```